src/sentry/static/sentry/app/utils/discover/styles.tsx

```diff
@@ -75,21 +75,22 @@ export const StyledShortId = styled(ShortId)`
   justify-content: flex-start;
 `;
 
-const StyledIconCopy = styled(IconCopy)`
+const IconWrapper = styled('span')`
   cursor: pointer;
-  margin-right: ${space(0.5)};
+  margin-left: ${space(1)};
+  color: ${p => p.theme.gray2};
 `;
 
 export const EventId = ({value}: {value: string}) => {
   const shortId = value.substring(0, 8);
   return (
     <React.Fragment>
+      {shortId}
       <Clipboard value={value}>
-        <span>
-          <StyledIconCopy size="xs" />
-        </span>
+        <IconWrapper>
+          <IconCopy size="xs" />
+        </IconWrapper>
```

Comment:
One problem with putting the icon on the right, is that the cell actions in discover rows will overlap this icon and make it hard to click it. Maybe we don't need the copy icon at all? Generally people will be able to use cell actions to update search conditions, or the event id will be a link as it is in performance views.

Reply:
Ohhh! good to know I'll revert it

Reply:
Actually yeah - let's remove the copy icon after all...